drivers/can/can_stm32.h

```diff
@@ -69,6 +69,7 @@ struct can_stm32_data {
 
 struct can_stm32_config {
 	CAN_TypeDef *can;   /*!< CAN Registers*/
+	CAN_TypeDef *filter_can;   /*!< CAN Registers for shared filter */
```

Comment:
I think we should call it master_can. It would be the term used in the datasheet.